src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1821,6 +1884,118 @@ public IndexColumn(string name, string @operator, string collation, SortOrder so
             public NullSortOrder NullSortOrder { get; }
         }
 
+        private static void ValidateTableDistributionProperties(
```

Comment:
This can be a local method at the end of Generate(CreateTableOperation), as it's not called by anyone else.

Also, `distributedByColumnName` doesn't actually seem to be check anywhere.